### ISSUE TITLE
Add feature aggregation to Disolve node

### ISSUE THREAD

**Issue by koettert** [spatial-data-lab/knime-geospatial-extension#284]
As a user I want to select the feature columns and a corresponding aggregation method for each individually.

Depends on AP-20920

**Comment by koettert** [comment]
Need to postpone this issue since the array parameter (AP-20920) are not yet supported in the Python framework.
